doc/releases/changelog-dev.md

```diff
@@ -98,6 +98,10 @@
 * ``qml.load`` has been removed in favour of more specific functions, such as ``qml.from_qiskit``, etc.
   [(#5654)](https://github.com/PennyLaneAI/pennylane/pull/5654)
 
+* ``qml.transforms.convert_to_numpy_parameters`` is now a proper transform and its output signature has changed,
```

Comment:
```suggestion
* `qml.transforms.convert_to_numpy_parameters` is now a proper transform and its output signature has changed,
```
changelog is markdown with single backticks.

Reply:
Will do! But notice that some items in that section also have double backticks